general-superstaq/general_superstaq/check/.isort.cfg

```diff
@@ -0,0 +1 @@
+../../.isort.cfg
```

Comment:
(this is necessary so `general-superstaq/general_superstaq/check/setup.cfg` doesn't override `general-superstaq/.isort.cfg` in this directory)